packages/tools/kda-cli/src/utils/spawn.ts

```diff
@@ -0,0 +1,14 @@
+import { spawn } from 'cross-spawn';
+
+export const spawned = (
+  command: string,
+  silent: boolean = false,
+): Promise<unknown> => {
+  return new Promise((resolve, reject) => {
+    const s = spawn('sh', ['-c', command], {
+      stdio: silent ? 'pipe' : 'inherit',
+    });
+    s.on('error', reject);
+    s.on('exit', resolve);
```

Comment:
Does this work on windows?

Reply:
In WSL yes, outside of WSL maybe with git bash installed or the like.

Reply:
Can we add a guard to assure that we're on a compatible OS and give a warning when we're not?